docs/en/writing-running-appium/tutorial/swipe/simple-screen.md

```diff
@@ -0,0 +1,70 @@
+## Simple Screen swipe
+
+Phone applications have interface. Interface, in most cases, blocks your
+swipe actions when you start from it. Historically it uses header and/or
+footer. To make screen swipe more reliable - it is better to start swipe
+action from center of screen.
+
+![swipe_screen](simple-screen.png)
+
+```java
+/**
+ * Performs screen swipe from center
+ *
+ * @param dir d - down, u - up, l - left, r - right
+ * @version java-client: 7.3.0
+ **/
+public void swipeScreen(String dir) {
```

Comment:
I'd rather make dir an enum

Reply:
we try to write code code as simple as possible. if we try try write all "clean code" style it will make harder to understand for most of appium beginners. all staff like: nice logs instead of println, exceptions, enums and so on everyone can make itself.

Reply:
> nice logs instead of println

agree, 

> exceptions, enums

catching generic exceptions, silencing them or not using enums does not make the code simpler, but only increases the count of possible issues in it and makes them harder to locate.